tests/integration/targets/aws_ses_identity_dkim/tasks/main.yml

```
@@ -0,0 +1,160 @@
+---
+# tasks file for aws_ses_identity_dkim test
+
+- name: set up aws connection info
+  set_fact:
+    aws_connection_info: &aws_connection_info
+      aws_access_key: "{{ aws_access_key }}"
+      aws_secret_key: "{{ aws_secret_key }}"
+      security_token: "{{ security_token }}"
+      region: "{{ aws_region }}"
+  no_log: yes
+
+- block:
```

It's better to use module_defaults rather than the facts hack.  AnsibleAWSModule defines some specific parameters as 'secret' and Ansible should *never* log them.

```suggestion
- name: 'Run tests with defaults'
  module_defaults:
    # Normally we'd use group/aws but this doesn't properly work for new module in collections yet
    aws_ses_identity:
        aws_access_key: "{{ aws_access_key }}"
        aws_secret_key: "{{ aws_secret_key }}"
        security_token: "{{ security_token | default(omit) }}"
        region: "{{ aws_region }}"
  block:
```